x-pack/plugins/observability/tsconfig.json

```diff
@@ -85,8 +85,10 @@
     "@kbn/deeplinks-analytics",
     "@kbn/observability-ai-assistant-plugin",
     "@kbn/osquery-plugin",
+    "@kbn/content-management-plugin",
+    "@kbn/embeddable-plugin",
+    "@kbn/react-kibana-mount",
```

Comment:
TODO: need to check why this was added

Reply:
Because of `import { toMountPoint } from '@kbn/react-kibana-mount';` in the [handle_explicit_input ](https://github.com/elastic/kibana/blob/91af1341800c17ba777400d4cd174b7bb7ce47de/x-pack/plugins/observability/public/embeddable/slo/handle_explicit_input.tsx)file